src/components/bulk-add/BulkAddPage.spec.js

```diff
@@ -30,23 +30,26 @@ describe('Bulk Add Page', () => {
     });
 
     describe('User scans an ISBN', () => {
+        let fakeIsbn;
         beforeEach(async () => {
+            fakeIsbn = (Math.random() * 100000000).toString();
+
             fetchMock.mockResponse(JSON.stringify({
-                'ISBN:0201634554': {
+                [`ISBN:${fakeIsbn}`]: {
                     title: 'Java'
                 }
             }));
 
-            await act(() =>  getScannerInputProps().onIsbnScanned('0201634554'));
+            await act(() =>  getScannerInputProps().onIsbnScanned(fakeIsbn));
             await wait();
         });
 
         it('should populate the ISBN field', () => {
-            expect(rendered.getByLabelText('ISBN')).toHaveValue('0201634554');
+            expect(rendered.getByLabelText('ISBN')).toHaveValue(fakeIsbn);
         });
 
         it('should attempt to fetch the book title', () => {
-            expect(fetchMock).toHaveBeenCalledWith('/api/books?bibkeys=ISBN%3A0201634554&jscmd=data&format=json', expect.anything());
```

Comment:
Since the fetch call has been refactored out into a shared utility module. Should we mock that module and move some of these tests into a test for that module?

Otherwise we are only testing that module as a side effect of this particular usage (even though it is being used elsewhere)

Reply:
I like this approach! I'll directly test that file. Do you have a suggestion for how to properly test the hook? Would it be sufficient to create a dummy component in the test file to mount the react hook on? I'll finish this cleanup tomorrow :+1:

Reply:
There's a companion library for react-testing-library: https://github.com/testing-library/react-hooks-testing-library

It has examples on how to unit test a hook.